src/ol/source/GeoTIFF.js

```diff
@@ -1017,6 +1017,14 @@ class GeoTIFFSource extends DataTile {
 
     return data;
   }
+
+  /**
+   * Return affine transformation matrix derived from the GeoTIFF ModelTransformation, if present.
+   * @return {import("../transform.js").Transform|null} Affine transformation matrix.
+   */
+  getTransformMatrix() {
+    return this.transformMatrix?.slice() || null;
+  }
```

Comment:
Since this is only used internally, it would be better to remove the `@protected` flag from `ol/source/DataTile#transformMatrix`, and get rid of this getter. On a related note, a getter returning a copy of something is not really a getter.